src/Product/SyncerHooks.php

```diff
@@ -82,39 +88,45 @@ class SyncerHooks implements Service, Registerable {
 	 * @param ProductHelper         $product_helper
 	 * @param JobRepository         $job_repository
 	 * @param MerchantCenterService $merchant_center
+	 * @param NotificationsService  $notifications_service
 	 * @param WC                    $wc
 	 */
 	public function __construct(
 		BatchProductHelper $batch_helper,
 		ProductHelper $product_helper,
 		JobRepository $job_repository,
 		MerchantCenterService $merchant_center,
+		NotificationsService $notifications_service,
 		WC $wc
 	) {
-		$this->batch_helper        = $batch_helper;
-		$this->product_helper      = $product_helper;
-		$this->update_products_job = $job_repository->get( UpdateProducts::class );
-		$this->delete_products_job = $job_repository->get( DeleteProducts::class );
-		$this->merchant_center     = $merchant_center;
-		$this->wc                  = $wc;
+		$this->batch_helper          = $batch_helper;
+		$this->product_helper        = $product_helper;
+		$this->update_products_job   = $job_repository->get( UpdateProducts::class );
+		$this->delete_products_job   = $job_repository->get( DeleteProducts::class );
+		$this->merchant_center       = $merchant_center;
+		$this->notifications_service = $notifications_service;
+		$this->wc                    = $wc;
 	}
 
 	/**
 	 * Register a service.
 	 */
 	public function register(): void {
 		// only register the hooks if Merchant Center is connected and ready for syncing data.
+		// TODO: Potentially change this after API Pull is implemented as we don't need MC to be connected for the API Pull
```

Comment:
I think we will still need the MC to be connected, but instead of pushing the data, they will pull the data.

Reply:
Mmm, I thought that authorizing the App and WPCOM Google can actually get data by themselves right?

Reply:
Yes, they will have access to the data however, from what I gather, they still require the MC ID, and the account needs to be connected.